modules/mcuboot/Kconfig

```diff
@@ -61,6 +61,23 @@ config MCUBOOT_FLASH_WRITE_BLOCK_SIZE
 
 endif # BOOTLOADER_MCUBOOT
 
+config BOOT_IMAGE_ACCESS_HOOK_CUSTOM
+	bool "Custom image access hooks"
+	depends on BOOT_IMAGE_ACCESS_HOOKS
+	help
+	  Enabling this option allows you to use a custom hooks file for the
+	  MCUBoot image access hooks. The same option needs to be set for the
+	  corresponding mcuboot child image.
+
+config BOOT_IMAGE_ACCESS_HOOK_CUSTOM_PATH
+	string "Path to hooks file"
+	default "nrf_custom_hooks.c"
```

Comment:
This is not an improvement, this makes things worse. This needs to be added *by cmake* and cmake only, in a way that allows people to overwrite it or add additional files to it if they want. Kconfigs should not have filenames in.